### ISSUE TITLE
Definition of 'department'

### ISSUE THREAD

**Issue by neilotte** [biobanking/biobanking#53]
The present definition of department holds: 'An organization with a specific responsibility that is a part of a larger organization with a specific responsibility.' 

Two issues:
a. Entities can be parts of themselves. Hence, this definition entails only that a department is an organization that bears a responsibility. This is too broad. 
b. It should be explicit that the responsibility of a department is distinct from the responsibility of the organization as a whole. 

Proposed revised definition: 'An organization that is a proper part of a parent organization, and which bears a specific responsibility distinct from that of its parent organization.'

**Comment by mbrochhausen** [comment]
a) is certainly correct.
b) I am not sure that is correct. Take the somewhat classic example of a university. The universities responsibility is to further education and research. The Department of Biology's responsibility is to further education and research in biology. I am not sure it is correct to say that those two responsibilities are distinct. The responsibility of the Department of Biology is more specific, but still not completely distinct, in my view.

**Comment by neilotte** [comment]
Perhaps there's a better way to put this, since I intend to address the sort of example you describe. By 'distinct', I mean the department and its parent do not share a responsibility, nor two responsibilities, that may be characterized in precisely the same manner. In other words, there must be a division of labor between the parent and its department, such that there is some delegating. So when the parent has a responsibility to further education and the department has a responsibility to further biology education, I take these two be distinct in this sense. Perhaps this is something for an elucidation as well.

**Comment by mbrochhausen** [comment]
Yes, I saw what you were getting at and I am in agreement with the fact that this needs to be addressed. 
I have a few further questions for clarification:
1. What are we thinking "responsibilities" are?
2. Do we agree that the parent organization's responsibilities and the child organization's responsibilities are somewhat aligned or are at least not counteracting (e.g. the parent organization is responsible to fight cancer and the child organization is responsible for promoting tobacco smoking).

**Comment by cstoeckert** [comment]
I disagree that a department does not share a responsibility with its parent. I think that any responsibility of a department is also responsibility of the parent but as a part the department does not share all the responsibilities of the parent organization. How about this as a proposed revised definition: 'An organization that is a proper part of a parent organization, and which bears a specified responsibility of its parent organization.'

**Comment by neilotte** [comment]
@cstoeckert Agreed on the first claim. I was too hasty in asserting that departments and their parents **can't** share a responsibility (Initially, I was reading 'responsibility' as denoting 'primary mission of the organization', and inferring that departments would bear unique specializations of the parent's primary mission. If we wished to go back to this, perhaps we could change the word 'responsibility' to something clearer, like _objective_). But even if by 'responsibility' we mean any old 'obligation', I should have stuck with the thesis that a department has some responsibility distinct from the set of responsibilities of its parent--a responsibility which may (and I affirm Mathias's question 2 above) be _in-line_ with the responsibilities of the parent organization--and allowed that departments and their parents can share other responsibilities (e.g. both could have a broad responsibility to respect patient rights in accordance with HIPAA). 

However, I don't think that if a department has a responsibility that we can infer that its parent shares that responsibility. In many cases, departments and their parents will share some responsibilities, but in many cases not (e.g. If a hospital administrator asks one of the hospital's departments to always make sure its members turn the lights off before going home, then the department has such a responsibility, but this isn't a responsibility of the hospital.).
